frontend/src/component/changeRequest/DraftBanner/DraftBanner.tsx

```diff
@@ -1,20 +1,71 @@
-import { useState, VFC } from 'react';
+import { FC, useState, VFC } from 'react';
 import { Box, Button, Typography } from '@mui/material';
 import { useStyles as useAppStyles } from 'component/App.styles';
 import WarningAmberIcon from '@mui/icons-material/WarningAmber';
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 import { ChangeRequestSidebar } from '../ChangeRequestSidebar/ChangeRequestSidebar';
 import { useChangeRequestOpen } from 'hooks/api/getters/useChangeRequestOpen/useChangeRequestOpen';
+import { IChangeRequest } from '../changeRequest.types';
 
 interface IDraftBannerProps {
     project: string;
 }
 
-export const DraftBanner: VFC<IDraftBannerProps> = ({ project }) => {
+const DraftBannerContent: FC<{
+    changeRequest: IChangeRequest;
+    onClick: () => void;
+}> = ({ changeRequest, onClick }) => {
     const { classes } = useAppStyles();
+
+    return (
+        <Box className={classes.content}>
+            <Box
+                sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    px: 1,
+                    py: 1.5,
+                    color: theme => theme.palette.warning.main,
+                }}
+            >
```

Comment:
Could be a styled component

Reply:
I was migrating the existing component and wanted to draw a line of how much to clean up in the process. But sure, I can also fix this one :)